src/components/organisms/ProfileModal/ProfilePopoverContent.scss

```diff
@@ -16,6 +16,7 @@ $profile-popover--margin-top: 12px;
   overflow: auto;
   padding: $spacing--md;
   background: $black;
+  height: 100%;
```

Comment:
can you explain why was that added? can't see the difference if I remove it. 🙈

Reply:
@sunny-viktoryia If you add too many user statuses it breaks, you can find the bug in the description of this PR.